src/pages/commits/ActiveCommits/CommitChartLabel.tsx

```diff
@@ -36,11 +58,31 @@ const LabelContainer = styled.div`
   display: flex;
   margin-top: 10px;
   flex-direction: column;
-  justify-content: flex-start;
   align-items: flex-start;
+  word-break: break-word;
 `;
 
-const Text = styled(Disclaimer)`
+const LabelText = styled(Body)`
   color: ${gray.dark2};
   width: 100%;
+  font-size: 12px;
+`;
+
+const ButtonContainer = styled.div`
+  cursor: pointer;
+`;
+
+const ButtonText = styled(Disclaimer)`
+  text-align: center;
+  color: ${blue.dark2};
+  text-decoration: underline;
+`;
+
+const TooltipContainer = styled(Body)`
+  width: 200px;
+  border-radius: 3px;
+  background-color: ${gray.light3};
+  color: ${gray.dark3};
+  word-break: break-word;
+  text-align: left;
 `;
```

Comment:
Is this styling necessary can we just use the default tooltip styles?

Reply:
You're right! Removed most of the styling.